shared/src/business/useCases/serveCaseToIrs/serveCaseToIrsInteractor.test.js

```diff
@@ -977,6 +981,45 @@ describe('serveCaseToIrsInteractor', () => {
     mockCase = {
       ...MOCK_CASE,
       orderForAmendedPetition: true,
+      receivedAt: '2018-03-01T21:40:46.415Z',
+    };
+
+    await serveCaseToIrsInteractor(applicationContext, {
+      docketNumber: MOCK_CASE.docketNumber,
+    });
+
+    expect(
+      await applicationContext.getUseCaseHelpers()
+        .addDocketEntryForSystemGeneratedOrder.mock.calls[0][0]
+        .systemGeneratedDocument,
+    ).toMatchObject({
+      content: expect.not.stringContaining('['),
+    });
+
+    const receivedDate = formatDateString(
+      '2018-03-01T21:40:46.415Z',
+      FORMATS.MONTH_DAY_YEAR,
+    );
+
+    const mockTodayPlus60 = getBusinessDateInFuture({
+      numberOfDays: 60,
+      startDate: formatNow(FORMATS.ISO),
+    });
+
+    expect(
+      await applicationContext.getUseCaseHelpers()
+        .addDocketEntryForSystemGeneratedOrder.mock.calls[0][0]
+        .systemGeneratedDocument,
+    ).toMatchObject({
+      content: `&nbsp;&nbsp;&nbsp;&nbsp;The Court filed on ${receivedDate}, a document as the petition of the above-named petitioner(s) at the docket number indicated. That docket number MUST appear on all documents and papers subsequently sent to the Court for filing or otherwise. The document did not comply with the Rules of the Court as to the form and content of a proper petition. <br/><br/>&nbsp;&nbsp;&nbsp;&nbsp;Accordingly, it is <br/><br/>&nbsp;&nbsp;&nbsp;&nbsp;ORDERED that on or before ${mockTodayPlus60}, petitioner(s) shall file a proper amended petition. If, by ${mockTodayPlus60}, petitioner(s) do not file an Amended Petition, the case will be dismissed or other action taken as the Court deems appropriate.`,
+    });
+  });
+
+  it('should replace brackets in orderForAmendedPetitionAndFilingFee content with the served date of the petition, and today plus sixty twice', async () => {
```

Comment:
Seems like some of these tests need to be updated to reflect the change from servedDate to filingDate?

Reply:
Hmmmm, yes.  Good catch.  Time for some spelunking to see why these didn't fail.

Reply:
@halprin, they don't fail because I did change the code but failed to change the `it` statement of what should happen.

Reply:
Done!